packages/datadog_webview_tracking/example/android/build.gradle

```diff
@@ -0,0 +1,39 @@
+buildscript {
+    ext.kotlin_version = '1.6.21'
+    repositories {
+        google()
+        mavenCentral()
+        maven {
+            url "https://plugins.gradle.org/m2/"
+        }
+    }
+
+    dependencies {
+        classpath 'com.android.tools.build:gradle:7.2.0'
```

Comment:
should it be `7.4.1` here? like in other sample apps.

Reply:
Yes probably. Thanks!